addons/delivery/models/delivery_grid.py

```diff
@@ -93,15 +93,15 @@ def _get_price_available(self, order):
                 continue
             if line.is_delivery:
                 total_delivery += line.price_total
```

Comment:
total_delivery is not used anymore after your changes, you could remove this part (and the var specification above)